<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1324</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModelLoader</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="ASHOpenglWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>991</width>
      <height>660</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="bg" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1321</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="shijiaotiaozheng" native="true">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>0</y>
       <width>331</width>
       <height>231</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>141</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;楷体&quot;;
font: 11pt &quot;Agency FB&quot;;
font: 16pt &quot;楷体&quot;;</string>
      </property>
      <property name="text">
       <string>视角调整</string>
      </property>
     </widget>
     <widget class="QWidget" name="rx" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <widget class="QSlider" name="Slider_xrot">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>180</width>
         <height>30</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>xrot</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="xrot">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>0</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="ry" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <widget class="QSlider" name="Slider_yrot">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>180</width>
         <height>30</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>yrot</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="yrot">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>0</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="rz" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <widget class="QSlider" name="Slider_zrot">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>180</width>
         <height>30</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>zrot</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="zrot">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>0</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="px" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <widget class="QSlider" name="Slider_xtrans">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>180</width>
         <height>30</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>xtrans</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="xtrans">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>0</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="py" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <widget class="QSlider" name="Slider_ytrans">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>180</width>
         <height>30</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>ytrans</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="ytrans">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>0</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pz" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <widget class="QSlider" name="Slider_ztrans">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>180</width>
         <height>30</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>ztrans</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="ztrans">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>0</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="showInput">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>250</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 170, 255);
font: 10pt &quot;Times New Roman&quot;;
font: 10pt &quot;Times New Roman&quot;;


color: rgb(255, 255, 255);
alternate-background-color: rgb(85, 170, 255);
border-color: rgb(85, 170, 255);
selection-background-color: rgb(85, 170, 255);
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>showInput</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showOut">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>305</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 170, 255);
font: 10pt &quot;Times New Roman&quot;;
font: 10pt &quot;Times New Roman&quot;;


color: rgb(255, 255, 255);
alternate-background-color: rgb(85, 170, 255);
border-color: rgb(85, 170, 255);
selection-background-color: rgb(85, 170, 255);
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>showOut</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showError">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>360</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 170, 255);
font: 10pt &quot;Times New Roman&quot;;
font: 10pt &quot;Times New Roman&quot;;


color: rgb(255, 255, 255);
alternate-background-color: rgb(85, 170, 255);
border-color: rgb(85, 170, 255);
selection-background-color: rgb(85, 170, 255);
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>showError</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showHelp">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>415</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 170, 255);
font: 10pt &quot;Times New Roman&quot;;
font: 10pt &quot;Times New Roman&quot;;


color: rgb(255, 255, 255);
alternate-background-color: rgb(85, 170, 255);
border-color: rgb(85, 170, 255);
selection-background-color: rgb(85, 170, 255);
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>showHelp</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>580</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>三维模型误差分析工具</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="bg2" native="true">
    <property name="geometry">
     <rect>
      <x>-7</x>
      <y>0</y>
      <width>1421</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/demo/4.png);</string>
    </property>
   </widget>
   <zorder>bg2</zorder>
   <zorder>bg</zorder>
   <zorder>openGLWidget</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1324</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="open_input"/>
    <addaction name="open_out"/>
    <addaction name="clear_all"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_input"/>
    <addaction name="action_output"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="action_error"/>
    <addaction name="color"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="about"/>
    <addaction name="help"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>其他</string>
    </property>
    <addaction name="show_window3"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <action name="open_input">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/demo/FolderOpened.png</normaloff>:/demo/FolderOpened.png</iconset>
   </property>
   <property name="text">
    <string>open_input</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="open_out">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/demo/OpenFolder.png</normaloff>:/demo/OpenFolder.png</iconset>
   </property>
   <property name="text">
    <string>open_out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="clear_all">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/demo/Run.png</normaloff>:/demo/Run.png</iconset>
   </property>
   <property name="text">
    <string>clear_all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionsave_T">
   <property name="text">
    <string>save_T</string>
   </property>
  </action>
  <action name="actionsave_path">
   <property name="text">
    <string>save_path</string>
   </property>
  </action>
  <action name="clear_txt">
   <property name="text">
    <string>clear_txt</string>
   </property>
  </action>
  <action name="read_path">
   <property name="text">
    <string>read_path</string>
   </property>
  </action>
  <action name="action_input">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/demo/GenerateFile.png</normaloff>:/demo/GenerateFile.png</iconset>
   </property>
   <property name="text">
    <string>展示input</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="action_output">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/demo/OpenFile.png</normaloff>:/demo/OpenFile.png</iconset>
   </property>
   <property name="text">
    <string>展示output</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="action_error">
   <property name="text">
    <string>计算误差</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+6</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+8</string>
   </property>
  </action>
  <action name="color">
   <property name="text">
    <string>色差表</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+7</string>
   </property>
  </action>
  <action name="help">
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+9</string>
   </property>
  </action>
  <action name="show_window3">
   <property name="text">
    <string>问题</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ASHOpenglWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">ashopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="demo.qrc"/>
 </resources>
 <connections/>
</ui>
